src/controllers/import.js

```diff
@@ -80,11 +80,17 @@ function ImportController(context) {
     }
   }
 
-  function validateImportApiKey(importApiKey) {
+  function validateImportApiKey(importApiKey, scopes) {
     // Parse the allowed import keys from the environment
     if (!allowedApiKeys.includes(importApiKey)) {
       throw new ErrorWithStatusCode('Invalid import API key', 401);
     }
+
+    try {
+      auth.checkScopes(scopes);
+    } catch (error) {
```

Comment:
I see the error isn't being logged here, but I think that's probably fine since a detailed log message is being logged by the `checkScopes` function.

```
    log.error(`API key with ID: ${authInfo.getProfile()?.api_key_id} does not have required scopes. It's missing: ${missingScopes.join(',')}`);
```